README.md

```diff
@@ -205,7 +205,7 @@ You will also need to provide a valid email to register your domain in LetsEncry
 The final output of running this module will not only be the creation of cloud resources but also, it will generate a valid `jx-requirements.yml` file that will be used by Jenkins X through `jx boot -r jx-requirements.yml`.
 The template can be found in: 
 
-https://github.com/jenkins-x/jx-cloud-provisioners/blob/master/eks/terraform/jx/jx-requirements.yml.tpl
+https://github.com/jenkins-x/terraform-aws-eks-jx/blob/master/jx/jx-requirements.yml.tpl
```

Comment:
What's about adding a TOC (table of content). The README is so long so that I think adding a TOC really helps. What editor are you using? I think there are plugins for Markdown TOC for pretty much any environment/editor.

Reply:
I also think something is wrong with your headings. Some of them don't render properly, see Vault, External DNS, ...
Not quite sure what the problem is. It looks ok, but even locally it won't render properly. When I then delete all '### ' and re-add it, then it works. Maybe some odd meta character?

Reply:
```
  terraform init
    terraaform apply -var 'cluster_name=<cluster_name>' -var 'region=<your_aws_region>' -var 'account_id=<your_aws_account_id>' 
```

There is a typo in 'terraaform'. Also, I think we should try to aim for the minimal setup using a module reference :

```
module "eks-jx" {
  source  = "jenkins-x/eks-jx/aws"
  
 account_id = "<my-account-id>"
}
```

We should be able to default cluster name and region.

Either way, we need to describe how to get to the step of `terraform init|apply`. Either by creating a _main.tf_ as above in an empty directory or in the case of the current example by cloning the repo.

Reply:
I think we should consistently fence code examples and not mix between  '```' and intendation

Reply:
You are mixing 'we' and 'you' when addressing the user. It should be consistent. I think 'you' is mainly used, but there is also:

> With this module, we can choose to create the Vault resources that will be used by Jenkins X.
 
On that note, what is the subject of "with this module"? 

Technically, do we really make Vault creation optional? Is the aim not to get away from local secret storage?